app/src/components/BookList/BookList.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import { Book } from '../Book/'
+
+import './BookList.css'
+
+export default class BookList extends Component {
+  constructor( props ) {
+    super( props )
+    this.state = {
+      books: [],
+      page: 0,
+      fetchURL: ''
+    }
+    this.getPage = this.getPage.bind( this )
+  }
+
+  componentDidMount() {
+    this.getPage(0)
+  }
+
+  getPage( page ) {
+    let fetchURL = this.props.fetchURL + page.toString()
+
+    fetch( fetchURL )
+      .then( response => response.json() )
+      .then( json => json.data )
+      .then( booksArray => {
+        let books = booksArray.map( ( book, index ) => {
+          return <Book key={ index } book={ book } />
+        })
+        this.setState({ books, fetchURL })
+      })
+  }
+
+  handleNextPage() {
+    let page = this.state.page
+    page++
+    this.setState({ page })
+    this.getPage( page )
+  }
+
+  handlePrevPage() {
```

Comment:
Something here smells kinda weird to me...
If the `page = 0` then you are going to then reassign page to 0 and then set the state.
So i feel like you just need the converse of this if-else.
So check for page > 0 and then do the else things.
```
if( page > 0 ) {
  this. setState( {page} )
  this.getPage( page )
}